lib/std/rand.zig

```diff
@@ -61,14 +61,30 @@ pub const Random = struct {
     }
 
     /// Returns a random value from an enum, evenly distributed.
-    pub fn enumValue(r: Random, comptime EnumType: type) EnumType {
+    pub inline fn enumValue(r: Random, comptime EnumType: type) EnumType {
+        return r.enumValueWithIndex(EnumType, usize);
+    }
+
+    /// Returns a random value from an enum, evenly distributed.
+    /// An index into an array of all possible values is generated using the specified type,
+    /// allowing for results to be independent of `usize`.
```

Comment:
And maybe we can link back here.
```suggestion
    /// allowing for results to be independent of `usize`.
    /// If that is not of importance, use `enumValue`.
```
Same below.